### PR TITLE
[samples] Add standalone build scripts

### PR DESCRIPTION
**Pull request by onabiull**

### CODE REVIEW

File: samples/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required( VERSION 2.6.2 FATAL_ERROR )
```
Comment (dvrogozh) [inline samples/CMakeLists.txt]: I don't think that's a good idea to go below cmake required by mediasdk itself. And we check 2.8.5 on our side, right?

File: samples/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+project( mediasdk-samples )
+
+option( CMAKE_MFX_HOME "Where is Media SDK?" OFF )
+option( CMAKE_VTUNE_HOME "Where is VTune?" OFF )
+
+option( ENABLE_ITT "Build targets with ITT instrumentation support (requires VTune)?" OFF )
+
+option( ENABLE_SW "Build SW versions of the targets?" OFF )
```
Comment (dvrogozh) [inline samples/CMakeLists.txt]: do we need this?

File: samples/build.pl [hunk 3]
```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/perl
```
Comment (dvrogozh) [inline samples/build.pl]: can we avoid this wrapper altogether, please? Just default configuration of cmake should be enough.

**Comment by onabiull** [comment]
It is just for MSS, to provide consistent experience with previous releases. This change won't appear in master.

**Pull request merged**